src/modules/item/utils.ts

```diff
@@ -8,3 +8,29 @@ export function isEditable(item: Item) {
   const data = item.data as WearableData
   return !item.rarity || !data.category
 }
+
+export function getBodyShapeType(item: Item) {
```

Comment:
Maybe this should be used to refactor ItemBadge? it's using a similar but slightly different logic to accomplish the same thing.

Reply:
done 👍